src/server/pegasus_server_impl.cpp

```diff
@@ -1496,62 +1498,67 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
     //          3, restore_dir is exist
     //
     bool db_exist = true;
-    auto path = ::dsn::utils::filesystem::path_combine(data_dir(), "rdb");
-    if (::dsn::utils::filesystem::path_exists(path)) {
+    auto rdb_path = ::dsn::utils::filesystem::path_combine(data_dir(), "rdb");
+    auto duplication_path = duplication_dir();
+    if (::dsn::utils::filesystem::path_exists(rdb_path)) {
         // only case 1
```

Comment:
Add detail description to show what does case 1 mean:
```
// case 1: we open the db that already exist
```

Reply:
Has comment in code start: https://github.com/apache/incubator-pegasus/pull/917/files/c07465e1ddb1831b0953bce1f71c64b234a5c686#diff-951c6edb3ffd9c5cea7460f0cbdb7ebc1bd792615bf4d8415379665a872db06eR1489